pkgs/development/interpreters/yex-lang/derivation.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, rustPlatform, pkgconfig, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "yex-lang-${version}";
+  version = "0.1.0";
```

Comment:
Seems you are actually tracking the latest commit, shall we use `unstable-2021-12-25` instead?

Reply:
On the version = ?